packages/api/endpoints/providers.js

```diff
@@ -116,6 +109,7 @@ async function post(req, res) {
       return res.boom.badRequest(error.message);
     }
     log.error('Error occurred while trying to create provider:', error);
+    log.error(`Error occurred with provider: ${JSON.stringify(postgresProvider)}`);
```

Comment:
I wasn't sure actually.  It'll be before a 500 error, so it'll be read only by an integrator in cloudwatch.  I just added it as more information since the previous error wouldn't be very useful.  Maybe I should explicitly put both there. Because if the translate fails from API, this will be empty, but if the error is in the writing of the postgresProvider then the ApiProvider might not have the information you need to see why it failed to write.

Reply:
1d4587ec44. I can remove if you don't like it

Reply:
That seems fine, more logging in this case can't hurt.